src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationProvider.java

```diff
@@ -0,0 +1,50 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright © 2021 Atruvia AG <opensource@atruvia.de> as contributor to the ODPi Egeria project. */
```

Comment:
Needs to be standard copyright (or we need to gain approval)
Applies to additional files below